### PR TITLE
Use ES6 syntax.

### PR DESCRIPTION
**Pull request by mattcollier**
@davidlehn interested in eliminating this file if it's not needed:
https://github.com/digitalbazaar/bedrock-angular-credential/blob/master/credential-routes.js

### CODE REVIEW

File: credential-directive.js [hunk 1]
```diff
@@ -57,7 +55,8 @@ function factory(
         ]).then(function(results) {
           updates.pop();
           if(updates.length === 0) {
-            var displayer = brCredentialService.getDisplayer(credential);
+            var displayer = brCredentialService.getDisplayer(
+              credential);
```
Comment (dlongley) [inline credential-directive.js]: Was this over 80 chars?

File: credential-directive.js [hunk 2]
```diff
@@ -68,7 +67,8 @@ function factory(
 
             // TODO: can this HTML be done in the view instead?
             var template;
-            if(scope.options && scope.options.display === 'simple') {
+            if(scope.options && scope.options.display ===
+              'simple') {
```
Comment (dlongley) [inline credential-directive.js]: Over 80 chars?

File: credential-service.js [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-define(['jsonld', 'lodash'], function(jsonld, _) {
-
-'use strict';
+import jsonld from 'jsonld';
+import _ from 'lodash';
 
 /* @ngInject */
-function factory($rootScope, brRefreshService, brResourceService, config) {
+export default function factory($rootScope, brRefreshService, brResourceService,
+  config) {
```
Comment (dlongley) [inline credential-service.js]: Drop one, drop 'em all

File: credential-service.js [hunk 4]
```diff
@@ -97,7 +94,7 @@ function factory($rootScope, brRefreshService, brResourceService, config) {
     // TODO: register and find displayers using priority
     // Possibly use displayer 'q' param like Accept?
     // May want to be able to override
-    types = _.reject(types, function(type) { return type === 'Credential'; });
+    types = _.reject(types, function(type) { return type === 'Credential';});
```
Comment (dlongley) [inline credential-service.js]: Weird: A space after the opening `{` but no space before the closing `}`? Should be consistent.

File: credential-viewer-component.js [hunk 5]
```diff
@@ -59,7 +54,7 @@ function Ctrl(
 
   init();
 
-  $scope.$watch(function() { return self.credential; }, function(credential) {
+  $scope.$watch(function() { return self.credential;}, function(credential) {
```
Comment (dlongley) [inline credential-viewer-component.js]: Inconsistent brace spacing

File: credentials-list-directive.js [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
-define(['angular', 'jsonld'], function(angular, jsonld) {
-
-'use strict';
+import angular from 'angular';
+import jsonld from 'jsonld';
 
 /* @ngInject */
-function factory($location, brAlertService, brCredentialService, config) {
+export default function factory($location, brAlertService, brCredentialService,
+  config) {
```
Comment (dlongley) [inline credentials-list-directive.js]: Drop one, drop em all

**Review by dlongley — CHANGES_REQUESTED** [review]

File: credential-directive.js (hunk 1 above)
Comment (mattcollier) [reply credential-directive.js]: fixed.

File: credential-directive.js (hunk 2 above)
Comment (mattcollier) [reply credential-directive.js]: fixed.

File: credential-service.js (hunk 3 above)
Comment (mattcollier) [reply credential-service.js]: fixed.

File: credential-service.js (hunk 4 above)
Comment (mattcollier) [reply credential-service.js]: fixed.

File: credential-viewer-component.js (hunk 5 above)
Comment (mattcollier) [reply credential-viewer-component.js]: fixed.

File: credentials-list-directive.js (hunk 6 above)
Comment (mattcollier) [reply credentials-list-directive.js]: fixed.

**Comment by mattcollier** [comment]
@dlongley ready for another pass.

**Review by dlongley — APPROVED** [review]

**Comment by davidlehn** [comment]
I'm not sure what to do about those unused routes.  This module and the idp code are still hooked together a bit it seems.  The idp is actually using the view html template from this module.  But each have their own credentials.html.  We should figure out if things have progressed enough that we could fix up this situation in some appropriate way.

**Comment by mattcollier** [comment]
@davidlehn the routes are commented out in the release version.  So I'm confused about why we can't just eliminate the file.

**Comment by davidlehn** [comment]
@mattcollier Looking at the history of that file I can't recall what was going on. Was the intent to remove those routes from this module eventually?  Or to re-enable them once something else was fixed?  Yeah the file could be removed now if the intent was to never use those routes here.  If you understand the various interacting modules and where things should go, then do what is best.

**Comment by mattcollier** [comment]
@davidlehn unfortunately I don't know the answers to those questions.

**Pull request merged**